### PR TITLE
♻️ Add startup logs to RUT

### PR DESCRIPTION
**Pull request by matusdrobuliak66**
## What do these changes do?
- ♻️ Add startup logs to RUT
- ♻️ Change .env-devel `_TRACING={}` to `_TRACING=null` 

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/process_message_running_service_setup.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ async def _subscribe_to_rabbitmq(app) -> str:
 def on_app_startup(app: FastAPI) -> Callable[[], Awaitable[None]]:
     async def _startup() -> None:
         with log_context(
-            _logger, logging.INFO, msg="setup resource tracker"
+            _logger, logging.INFO, msg="RUT process message running service startup.."
```
Comment (mrnicegyu11) [inline services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/process_message_running_service_setup.py:35]: i dont get that string, what is "process message running service"

Comment (mrnicegyu11) [reply services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/process_message_running_service_setup.py:35]: the logline before was lear to me ;)

**Review by mrnicegyu11 — COMMENTED** [review]
thx

File: .env-devel [hunk 2]
```diff
@@ -389,6 +389,6 @@ WEBSERVER_SOCKETIO=1
 WEBSERVER_STATICWEB={}
 WEBSERVER_STUDIES_DISPATCHER={}
 WEBSERVER_TAGS=1
-WEBSERVER_TRACING={}
+WEBSERVER_TRACING=null
```
Comment (pcrespov) [inline .env-devel:392]: thx!

**Review by pcrespov — APPROVED** [review]

File: services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/process_message_running_service_setup.py (hunk 1 above)
Comment (matusdrobuliak66) [reply services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/services/process_message_running_service_setup.py]: improved

**Pull request merged**